/**:
  dvl_odom_converter_node:
    ros__parameters:
      input_topic: "dvl/position"
      output_topic: "odometry/dvl"

      publish_local_tf: false # Whether to publish the dvl_odom->base_link transform

  gps_odom_converter_node:
    ros__parameters:
      gps_odom_topic: "odometry/gps"
      ahrs_topic: "imu/data"
      output_topic: "odometry/truth"

      max_position_covariance: 0.01 # [m^2]
